### PR TITLE
docs: Revisit docs constructuons to Github markdown extension

### PR DESCRIPTION
**Pull request by MaxymVlasov**
Implementing https://github.com/orgs/community/discussions/16925

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -141,7 +139,7 @@ Set `-e PRE_COMMIT_COLOR=never` to disable the color output in `pre-commit`.
 <details><summary><b>MacOS</b></summary><br>
 
 ```bash
-brew install pre-commit terraform-docs tflint tfsec trivy checkov terrascan infracost tfupdate minamijoyo/hcledit/hcledit jq
+brew install pre-commit terraform-docs tflint tfsec trivy checkov terrascan infracost tfupdate minamijoyo/hcledit/hcledit jq flock
```
Comment (MaxymVlasov) [inline README.md:142]:
```suggestion
brew install pre-commit terraform-docs tflint tfsec trivy checkov terrascan infracost tfupdate minamijoyo/hcledit/hcledit jq
```

**Review by antonbabenko — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -88,6 +84,7 @@ If you are using `pre-commit-terraform` already or want to support its developme
 * [`jq`](https://github.com/stedolan/jq) required for `terraform_validate` with `--retry-once-with-cleanup` flag, and for `infracost_breakdown` hook.
 * [`tfupdate`](https://github.com/minamijoyo/tfupdate) required for `tfupdate` hook.
 * [`hcledit`](https://github.com/minamijoyo/hcledit) required for `terraform_wrapper_module_for_each` hook.
+* [`flock`](https://man7.org/linux/man-pages/man1/flock.1.html) highly recommended for `terraform_validate` and `terraform_providers_lock`
```
Comment (yermulnik) [inline README.md:87]: Wouldn't it make sense to move this one into corresponding PR? 🤔

**Review by yermulnik — APPROVED** [review]
👍🏻

File: README.md (hunk 2 above)
Comment (MaxymVlasov) [reply README.md:87]: Oops
```suggestion
```

**Comment by antonbabenko** [comment]
This PR is included in version 1.86.1 :tada:

**Pull request merged**